devtools/README.md

```diff
@@ -45,13 +45,16 @@ How to do a release
   Conda binaries should be automatically built with the `rc` tag.
 - To test the release candidate, you can create a virtual environment like this:
   `conda create -n msmb-test -c omnia/label/rc msmbuilder`, then run:
-  `source activate msmb-test`,
+  `source activate msmb-test`.
+  If you do not have the test dependencies listed [here](https://github.com/msmbuilder/msmbuilder/blob/e6839bccd3f1c49f0d1132375a95de61fd0914be/devtools/conda-recipe/meta.yaml), you will need
```

Comment:
or link to master instead of a specific commit? idk is there a reason why you wanted this commit?